README.md

```diff
@@ -384,6 +384,19 @@ repo.  Each commit is modified to make changes relative to the package
 directory.  So, for example, the commit that added `package.json` will
 instead add `packages/<directory-name>/package.json`.
 
+### export
+
+```sh
+$ asini export <package-name> --to <path-to-external-directory>
```

Comment:
The `<path-to-external-directory>` is a second argument now, right?  No `--to` required?